Automattic/jetpack
# VideoPress: Loading state on poster image update

Follow-up to https://github.com/Automattic/jetpack/pull/26762
Fixes #26500

#### Changes proposed in this Pull Request:

* Extracts two new actions `SET_UPDATING_VIDEO_POSTER` and `UPDATE_VIDEO_POSTER` and creates the video meta state `isUpdatingPoster`.
* Adds the update poster from library functionality to the edit details page

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard
* Use the quick actions of a video to update the poster image with an uploaded image. It should work normally.
* Go to the edit details page
* Update the poster image with an uploaded image. Do not save. Go back to the dashboard and check that the poster image was not updated
* Repeat the tests for the frame selection poster update

https://user-images.githubusercontent.com/8486249/195688771-a0f1790f-be72-4ec1-8251-9683696c654c.mp4
